modules/unsupported/mbtiles/src/test/java/org/geotools/mbtiles/MBTilesFileVectorTileTest.java

```diff
@@ -0,0 +1,68 @@
+package org.geotools.mbtiles;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.hamcrest.CoreMatchers.notNullValue;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertThat;
+
+import java.io.File;
+import java.io.InputStream;
+
+import org.apache.commons.io.IOUtils;
+import org.geotools.mbtiles.MBTilesMetadata.t_format;
+import org.geotools.mbtiles.MBTilesMetadata.t_type;
+import org.geotools.referencing.CRS;
+import org.junit.Before;
+import org.junit.Test;
+
+public class MBTilesFileVectorTileTest {
+    
+    File dbfile;
+    
+    @Before
+    public void setUp() {
+        dbfile = new File(MBTilesFileVectorTileTest.class.getResource("planet.mbtiles").getFile());
```

Comment:
@smithkm this bad URL->File conversion breaks the build in a path with spaces. The failure causes attempts to access filesystem paths containing ```%20``` that do not exist. Building in a path with spaces is a proxy for testing that code supports i18n-safe URL<->File conversions.

Fixed on master; please include this fix in any backport:
https://github.com/geotools/geotools/commit/d3d878b9a8cf9161b0a91d1b61b47ac8b06b68fb

Please use DataUtilities for all URL<->File conversions:
http://docs.geotools.org/latest/developer/conventions/code/url.html